internal/olm/operator/bundle/install.go

```diff
@@ -87,6 +109,65 @@ func (i *Install) setup(ctx context.Context) error {
 		return err
 	}
 
+	// get index image labels.
+	catalogLabels, err := registryutil.GetImageLabels(ctx, nil, i.IndexImageCatalogCreator.IndexImage, false)
+	if err != nil {
+		return fmt.Errorf("get index image labels: %v", err)
+	}
+
+	// set the field to true if FBC label is on the image or for a default index image.
+	if _, hasFBCLabel := catalogLabels[containertools.ConfigsLocationLabel]; hasFBCLabel || i.IndexImageCatalogCreator.IndexImage == registry.DefaultIndexImage {
+		i.IndexImageCatalogCreator.HasFBCLabel = true
+	}
```

Comment:
@rashmigottipati I just realized that we are not actually outputting any logs related to the deprecation of the SQLite index. All the opm-related warnings get written out to the registry pod logs, which may/may not be seen by the operator author. It might be valuable to add a warning statement as an else block here saying "The use of SQLite index images is deprecated and will be completely removed in a future release, please migrate your catalogs to the new File-Based Catalog format" so that a user is notified of the transition every time run bundle is invoked with a SQLite index image.

Reply:
That's right. I realized that opm related warnings get written out to registry pod logs instead. Added a deprecation log for SQLite index images in my latest commit.